src/RoomsSDKAdapter.js

```diff
@@ -126,12 +135,56 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
       );
 
       // Convert to a multicast observable
-      this.getRoomObservables[ID] = getRoom$.pipe(
-        publishReplay(1),
-        refCount(),
-      );
+      this.getRoomObservables[ID] = getRoom$.pipe(publishReplay(1), refCount());
     }
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits current and future activities from the specified room.
+   *
+   * @param {string} id ID of room to get
```

Comment:
Keep consistency with the rest of the repo
```suggestion
   * @param {string} ID ID of room to get
```